modules/createBowerPackage.js

```
@@ -0,0 +1,55 @@
+import { join as joinPaths} from 'path'
+import { stat as statFile, readFile, createWriteStream } from 'fs'
+import getProperty from './getProperty'
+import archiver from 'archiver'
+
+
+const promisify = func => (...args) =>
```

I've tried the tools. Lots of them. I have no doubt that support for swallowed exceptions will continue to get better over the coming years now that they've made it into ES6, which is great for people who want to use them. For me, personally, I've decided to avoid them if I can. Just feels like one place I can avoid some unnecessary complexity.